templates/partials/_what-is-juju.html

```diff
@@ -0,0 +1,137 @@
+        <img src="https://assets.ubuntu.com/v1/7b84cf45-what-is-juju-illustration-5.svg" alt="" class="p-feature__multi-8"/>
+        <img src="https://assets.ubuntu.com/v1/7b84cf45-what-is-juju-illustration-5.svg" alt="" class="p-feature__multi-9"/>
+      </div>
+    </div>
+  </div>
+  <div class="row u-full-height u-vertically-center">
+    <div class="col-3 p-heading-highlight">
+      <h3 class="p-heading-highlight__item p-heading--two">2.</h3>
+      <hr class="is-dark">
+      <p>Juju uses Charmed Operators ("Charms") and a Charmed Operator Lifecycle Manager to take control of the deployment, upgrades, integrations, management, and operations of those workloads across your hybrid cloud.</p>
+    </div>
+    <div class="col-9 u-hide--x-large">
+      <img src="https://assets.ubuntu.com/v1/760ba37a-what-is-juju-illustration-2.svg" alt=""/>
+    </div>
+  </div>
+  <div class="row u-full-height u-vertically-center">
+    <div class="col-3 p-heading-highlight">
+      <h3 class="p-heading-highlight__item p-heading--two">3.</h3>
+      <hr class="is-dark">
+      <p>Charms are small applications which package common maintenance functions, to turn Day 0 to Day 2 operations into repeatable and reliable code.</p>
+      <p>This enables your ops team to manage applications and scenarios rather than fixating on configurations (although they can dive into YAML whenever they like).</p>
+      <p><a class="p-button--positive" href="https://charmhub.io">See Charms on charmhub.io</a></p>
+    </div>
+    <div class="col-9 u-hide--x-large">
+      <img src="https://assets.ubuntu.com/v1/d7416dc8-3_full-panel_new.svg" alt=""/>
+    </div>
+  </div>
+  <div class="row u-full-height u-vertically-center">
+    <div class="col-3 p-heading-highlight">
+      <h3 class="p-heading-highlight__item p-heading--two">4.</h3>
+      <hr class="is-dark">
+      <p>With Charmed Operators, you not only deploy or manage individual applications, but you relate them to one another in "models", to handle scaling, management, and cross-service dependencies. Your application model defines which applications provide a service and how they interrelate.</p>
+    </div>
+    <div class="col-9 u-hide--x-large">
+      <img src="https://assets.ubuntu.com/v1/200de6bb-what-is-juju-illustration-4.svg" alt=""/>
+    </div>
+  </div>
+  <div class="row u-full-height u-vertically-center">
+    <div class="col-3 p-heading-highlight">
+      <h3 class="p-heading-highlight__item p-heading--two">5.</h3>
+      <hr class="is-dark">
+      <p>Models, cross-model relations, and model driven operations give you the control to handle deployments and operations at scale, across hybrid clouds, via CLI, or in a visual tool such as the Juju GUI or JAAS.</p>
+      <p><a class="p-button--positive" href="/docs">Check out the docs</a></p>
+    </div>
+    <div class="col-9 u-hide--x-large">
+      <img src="https://assets.ubuntu.com/v1/a87fb8bf-5_full-panel_new.png" alt=""/>
```

Comment:
can you not use the image template?

Reply:
I have updated the images that didn't break it